ci/timeout_with_stack.py

```
@@ -0,0 +1,353 @@
+        bt_command = "thread apply all bt"
+        print(f"\nCapturing C stack trace for process {pid}:")
+    else:
+        bt_command = "thread apply all py-bt"
+        print(f"\nCapturing Python stack trace for process {pid}:")
+    gdb = shutil.which("gdb")
+    if gdb is None:
+        print(f"Skipping stack trace for process {pid}: gdb not found")
+        return
+
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)